mythril/analysis/modules/dos.py

```diff
@@ -4,6 +4,7 @@
 from typing import Dict, cast, List
 
 from mythril.analysis.swc_data import DOS_WITH_BLOCK_GAS_LIMIT
+from mythril.laser.ethereum.strategy.custom import JUMPDEST_LIMIT
```

Comment:
Analysis code should not be dependent on search strategies, keeping them decoupled makes maintenance & further development easier.

Reply:
Then let's have a constants file for the laser module.That might be cool.

Reply:
There doesn't need to be a dependency here

Reply:
The JUMPDEST_LIMIT is dependent in both areas right? so this should have a dependece with laser.

Reply:
Yeah, the idea here is to make sure that the limit doesn't matches the loop bound because otherwise the it will never be triggered. But we can remove the dependency.

Reply:
Actually, now that the search strategy sets a `JumpdestCountAnnotation` this module could be simplified further by accessing that annotation instead of maintaining its own counter. However this would mean that the dependence remains.

Reply:
No, that's actually BS, because a different search strategy could be used.